docs/_sass/_docs-display-component.scss

```diff
@@ -50,16 +50,22 @@
     }
 
     .fd-date-picker {
-        & .fd-popover {
+        .fd-popover {
             &__body {
                 padding: 0 12px;
             }
         }
     }
 
+    .fd-time-picker {
+        .fd-popover__body {
+            padding: 16px;
```

Comment:
Following suit of Date Picker:
- add padding here for docs 
- remove the time picker scss itself